### DISCUSSION TITLE
CSS stopped loading

### DISCUSSION THREAD

**Discussion by goldmerc** [laravel/nova-issues#5314] [Q&A]
Nova Version: 3.32.0
Laravel Version: 9.51.0

I'm a bit stumped. My custom css has stopped loading. I've been doing lots of work on my site and I can't find what I did to mess it up. In my NovaServiceProvider boot()...

`Nova::style('custom-fields-css',` public_path('css/admin/custom.css'));`

which is auto generating in my html...

`<link rel="stylesheet" href="/nova-api/styles/custom-fields-css">`

but that just gets a 404.

Can anyone point me in the right direction?

**Comment by crynobone** [comment]
You should use `Nova::remoteStyle()`

**Comment by goldmerc** [comment]
Thanks @crynobone but it still didn't load.

I figured it out. For the frontend of my site, I had added a route with a {style} param and added a pattern restriction to it. So, nova's api was not matching the style rule. Removing my route pattern for style fixed it. So, I'll rename my param so it doesn't collide with Nova.

Out of interest, why do you recommend using remoteStyle over style? I can see that it does an md5 hash of the file path to use as the name. What's the benefit please?

Thanks for your help.

**Comment by goldmerc** [comment]
To clarify, I had added a global route pattern constraint on 'style'. I changed it to be applied directly to my route and now it doesn't interfere with nova's style route parameter
